src/core_modules/capture-core/components/Pages/New/NewPage.component.js

```diff
@@ -135,6 +83,24 @@ const NewPagePlain = ({
                     </Button>
                 </>
             }
+
+            {
+                newPageStatus === newPageStatuses.WITHOUT_PROGRAM_CATEGORY_SELECTED &&
+                (() => {
+                    const missingCategories = missingCategoriesInProgramSelection.reduce((acc, { name }, index) => {
+                        if ((index + 1 === missingCategoriesInProgramSelection.length)) {
+                            return `${acc} ${name} ${missingCategoriesInProgramSelection.length > 1 ? 'categories' : 'category'}`;
+                        }
+                        return `${acc} ${name},`;
+                    }, '');
+
+                    return (
+                        <InefficientSelectionsMessage
```

Comment:
NIT: Maybe Insufficient/Incomplete would be better?

Reply:
Yep good point. I am changing it